lib/drizzlepac/createMedian.py

```diff
@@ -169,22 +169,22 @@ def _median(imageObjectList, paramDict):
 
         if lthresh is not None and proc_units.lower() == 'native':
             if native_units.lower() == 'counts':
-                lthresh = lthresh * det_gain
+                lthresh *= det_gain
             elif native_units.lower() == 'counts/s':
-                lthresh = lthresh * det_gain * img_exptime
+                lthresh *= det_gain * img_exptime
             elif native_units.lower() == 'electrons/s':
-                lthresh = lthresh * img_exptime
+                lthresh *= img_exptime
             else:
                 raise ValueError("Unexpected native units: '{}'"
                                  .format(native_units))
 
         if hthresh is not None and proc_units.lower() == 'native':
             if native_units.lower().startswith('counts'):
```

Comment:
Using `startswith('counts')` here means that the test on 'counts/s' will never be reached.